Assets/Scripts/KatamariMovement.cs

```diff
@@ -440,6 +438,10 @@ public void UpdateTimeStateMachine()
 
     public virtual void OnCollisionStay(Collision collision)
     {
+        // DrawTrajectory simulation may run this before Start() is called, so the RigidBody
```

Comment:
Wow, I cant remember if we solved the timer not going up issue or not, but if not, does this improve it??

Reply:
Ya, this problem should be solved by making the sleep threshold of the katamari rb zero.